src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -966,6 +966,14 @@ def service_name(self) -> str:
             return f'{daemon_type_to_service(self.daemon_type)}.{self.service_id()}'
         return daemon_type_to_service(self.daemon_type)
 
+    def get_host_placement(self) -> HostPlacementSpec:
+        return HostPlacementSpec(
+            hostname=self.hostname or '',
+            # FIXME: include the ip:port here?
+            network='',
+            name='',
```

Comment:
In case we already have HostPlacementSpecs in the ServiceSpec, we should get that info from there instead.